src/Models/Services/Utils.ts

```diff
@@ -831,3 +832,30 @@ export const getMockTimeSeriesDataArrayInLocalTime = (
         })).sort((a, b) => (a.timestamp as number) - (b.timestamp as number));
     });
 };
+
+export function formatNumber(val: number) {
```

Comment:
can we add a block comment before this method to explain how it formats the passed number with e.g. input and output?